tornado-drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/OpenCL.java

```diff
@@ -171,11 +171,11 @@ public static void run(Long executionContextId, OCLTornadoDevice tornadoDevice,
             switch (access) {
                 case READ_WRITE:
```

Comment:
When using `READ_WRITE` accessor, then the OpenCL flag should be `READ_WRITE` too.